### ISSUE TITLE
eslint comment description in style block

### ISSUE THREAD

**Issue by donald-lendit** [future-architect/eslint-plugin-vue-scoped-css#245]
I saw in the documentation that eslint comments allow additional comments via `--`.

https://eslint.org/docs/latest/user-guide/configuring/rules#comment-descriptions

```
<template>
  <div>Content</div>
</template>

<script>
/* some javascript css class manipulation */
</script>

<style lang="scss" scoped>
// eslint-disable-next-line vue-scoped-css/require-selector-used-inside -- BUT THIS MAKES DISABLE RULE NOT WORK
.fixed-height {
  height: calc(100vh);
}
</style>

```

But adding a string with '--' eslint-disable comment doesn't work.
Am I misunderstanding something?
```
"eslint": "^8.28.0",
"eslint-plugin-vue-scoped-css": "^2.2.0",
```

**Comment by ota-meshi** [comment]
Thank you for this report.

Comments in CSS are not known to ESLint core, so comments are handled in this plugin.
And it still doesn't support `-- description`. We need to add a feature to this plugin.
